qt/build_style/build_statistics.cpp

```diff
@@ -22,16 +22,12 @@ QString GetStyleStatistics(QString const & mapcssMappingFile, QString const & dr
   if (!QFile(drulesFile).exists())
     throw std::runtime_error("drawing-rules file does not exist at " + drulesFile.toStdString());
 
-  // Add path to the protobuf EGG in the PROTOBUF_EGG_PATH environment variable.
```

Comment:
Why EGG was used before?

Reply:
This mechanism was employed to assist the designer in scenarios where the Python protobuf dependency might not be present, or to guarantee that the designer is utilizing the same protobuf version as OM.

I endeavored to update it to the current version; however, EGG files have been deprecated in Python, and protobuf no longer offers them.

Reply:
Does it mean that Designer won't work after merging this change?